### DISCUSSION TITLE
None of the UI elements are updating. Installation works perfectly on Dawn but not on client's store.

### DISCUSSION THREAD

**Discussion by juan-torre-dev** [EvgeniyMukhamedjanov/liquid-ajax-cart#21] [Q&A]
I have installed the library on Dawn local environment and it works perfectly. But when attempting the exact same installation on the client's store none of the Cart UI elements update. Any update to the cart is visible only after reloading the page. I've been stuck on this for several days now. I guess some code on the store may be conflicting with the library but not sure where to look. I don't know what else to try. Any help is appreciated. Thank you!

Edit: Upon further inspection I am not seeing the 'Sections are updated: ' log on the console that should log when manipulating the cart section.

I do have the script tag with the function subscribeToCartSectionsUpdate() in theme.liquid and am using the exact my-cart.liquid section provided on the installation guide.

Edit 2: Ok I forgot to mention this was all being done on cli shopify theme serve store. I tried doing it on the customizer link rather than the local preview and it appears to be working fine on there...

**Comment by EvgeniyMukhamedjanov** [comment]
Hey @JayP-TA can you share a preview link to the client store, I will check what is going on there.

**Reply by juan-torre-dev** [reply]
Hi @EvgeniyMukhamedjanov of course! 

I think you'll find it working fine. The issue here was that the script was not updating the HTML elements on the shopify theme serve store. And I never thought to upload and try the shopify admin preview beacuse on dawn it did work on shopify theme serve store.

Anyhow, here's the link: https://6ngilzvlww1cn5de-60892512476.shopifypreview.com

The library is super awesome btw if I hadn't run into this issue I would have done this in minutes!
